composer.json

```diff
@@ -16,7 +16,7 @@
         "php": ">=5.5",
         "ext-curl": "*",
         "guzzlehttp/guzzle": "~5.1",
-        "symfony/console": "~2.6"
+        "symfony/console": "~3.2"
```

Comment:
We must continue to have compatibility with lower versions. I suggest the following configuration:

```json
"symfony/console": "~2.6 || ~3.2"
```

Can you test this one?